File: problems/easy/sum_two/__tests__/test_sum_two_numbers.cpp
#include <cassert>
#include <iostream>
#include "../sum_two_numbers.cpp"

int main() {
    // Test 1
    assert(sumTwoNumbers(3, 4) == 7);
    // Test 2
    assert(sumTwoNumbers(0, 0) == 0);
    // Test 3
    assert(sumTwoNumbers(-5, 5) == 0);

    std::cout << "✅ All C++ tests passed." << std::endl;
    return 0;
}

File: problems/easy/sum_two/__tests__/test_sum_two_numbers.java
package problems.easy.sum_two.__tests__;

import problems.easy.sum_two.sum_two_numbers;

public class test_sum_two_numbers {
    public static void main(String[] args) {
        if (sum_two_numbers.sum_two(3, 4) != 7) {
            throw new AssertionError("Test 1 failed");
        }
        if (sum_two_numbers.sum_two(0, 0) != 0) {
            throw new AssertionError("Test 2 failed");
        }
        System.out.println("✅ All Java tests passed!");
    }
}
